### PR TITLE
Remove all beta properties and parameters from the Issuing API

### PR DESCRIPTION
**Pull request by remi-stripe**
This PR removes all the deprecated features associated with Issuing's beta before the GA happened. It also cleans up naming of various classes to mirror our current logic based on the openapi spec to ensure we will codegen the information reliably.

r? @richardm-stripe
cc @stripe/api-libraries

### CODE REVIEW

**Comment by ob-stripe** [comment]
@remi-stripe Same, should we close this for now?

**Comment by remi-stripe** [comment]
I'd like to keep the PR but I changed the title to be clear it's on hold until later this week

**Comment by ob-stripe** [comment]
I've reset the `integration-v36` branch, so you'll need to rebase to remove the extra commits.

File: src/Stripe.net/Entities/Issuing/Transactions/Transaction.cs [hunk 1]
```diff
@@ -7,23 +7,39 @@ namespace Stripe.Issuing
...
+        /// </summary>
         [JsonIgnore]
         public string AuthorizationId
         {
             get => this.InternalAuthorization?.Id;
             set => this.InternalAuthorization = SetExpandableFieldId(value, this.InternalAuthorization);
         }
 
+        /// <summary>
+        /// (Expanded)) The <see cref="Authorization"/> associated with this transaction.
```
Comment (richardm-stripe) [inline src/Stripe.net/Entities/Issuing/Transactions/Transaction.cs:41]: Should only be one close paren here, I think

File: src/Stripe.net/Entities/Issuing/Transactions/Transaction.cs [hunk 2]
```diff
@@ -37,13 +53,20 @@ public Authorization Authorization
...
+        /// </summary>
         [JsonIgnore]
         public string BalanceTransactionId
         {
             get => this.InternalBalanceTransaction?.Id;
             set => this.InternalBalanceTransaction = SetExpandableFieldId(value, this.InternalBalanceTransaction);
         }
 
+        /// <summary>
+        /// (Expanded)) The <see cref="BalanceTransaction"/> associated with this transaction.
```
Comment (richardm-stripe) [inline src/Stripe.net/Entities/Issuing/Transactions/Transaction.cs:68]: s/(Expanded))/(Expanded)

File: src/Stripe.net/Entities/Issuing/Cardholders/Cardholder.cs [hunk 3]
```diff
@@ -19,15 +19,11 @@ public class Cardholder : StripeEntity<Cardholder>, IHasId, IHasMetadata, IHasOb
...
-        [Obsolete("Use SpendingControls instead.")]
-        [JsonProperty("authorization_controls")]
-        public CardholderAuthorizationControls AuthorizationControls { get; set; }
-
         /// <summary>
         /// The cardholder’s billing address.
         /// </summary>
         [JsonProperty("billing")]
-        public Billing Billing { get; set; }
+        public CardholderBilling Billing { get; set; }
```
[the remark this answers is not in the document]
Comment (remi-stripe) [reply src/Stripe.net/Entities/Issuing/Cardholders/Cardholder.cs:26]: we decided to mirror java and the spec so this change is approved

**Comment by remi-stripe** [comment]
Fixed, waiting for tests to pass to be safe and then merging

**Pull request merged**